alvr/xtask/src/version.rs

```diff
@@ -0,0 +1,110 @@
+use crate::*;
+use alvr_common::data::{ALVR_CLIENT_VERSION, ALVR_SERVER_VERSION};
+
+use semver::{Version, VersionReq};
+use gradle_sync::BuildGradleFile;
+use toml_edit::Document;
+use std::str::FromStr;
+use std::fs::File;
+use std::cmp::Ordering;
+
+fn bumped_versions(server_version: &Option<String>, client_version: &Option<String>) -> Result<(Version, Version), Box<dyn Error>>{
```

Comment:
It's more idiomatic to use `Option<&T>` instead of `&Option<T>`. In the calling site then you would use `option.as_ref()`. It's just a nitpick, nothing serious :)
Also you could move the BResult alias in this file so you can use it here.

Reply:
Option<&str> is even better. I didn't know about option.as_deref(). Nice!